pkg/apis/openebs.io/scp/v1alpha1/affinity.go

```diff
@@ -0,0 +1,42 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Affinity is a group of affinity scheduling rules
+type Affinity struct {
+
+	// Describes volume affinity scheduling rules
+	// +optional
+	VolumeAffinity []VolumeAffinityTerm `json:"volumeAffinity,omitempty"`
+
+	// Describes volume anti-affinity scheduling rules
+	// +optional
+	VolumeAntiAffinity []VolumeAffinityTerm `json:"volumeAntiAffinity,omitempty"`
+
+	// Describes cohort affinity scheduling rules
+	// +optional
+	CohortAffinity []metav1.LabelSelector `json:"cohortAffinity,omitempty"`
+
+	// Describes cohort anti-affinity scheduling rules
+	// +optional
+	CohortAntiAffinity []metav1.LabelSelector `json:"cohortAntiAffinity,omitempty"`
```

Comment:
Using `CohortAffinity` we can avoid scheduling a volume in a group of storage cohort.

Reply:
Please update the comment with this.

Reply:
also add these are ANDed.